apps/aehttp/src/aehttpc_aeternity.erl

```diff
@@ -7,9 +7,6 @@
 -export([get_latest_block/2,
```

Comment:
My feeling is that this whole module can be deleted if we don't care about commitments. What else did it offer that is not in the standard http client software??

Reply:
For similarity to bitcoin and dodge module, we could keep it, of course. We still need to be able to see what epoch the parent chain is in

Reply:
It's just a wrapper but since it is a wrapper that gets dynamically selected (ae, btc, doge) there needs to be one for aeternity as well. I kind-of like this way, so I think it should be kept.